logplex.go

```diff
@@ -1,44 +1,34 @@
 package hsup
 
 import (
-	"bufio"
-	"fmt"
+
 	"io"
-	"net/http"
 	"net/url"
-	"time"
 
-	"github.com/logplex/logplexc"
+	shuttle "github.com/heroku/log-shuttle"
+    "io/ioutil"
 )
 
 type relay struct {
-	cl   *logplexc.Client
+	cl   *shuttle.Shuttle
 	name string
 }
 
 func newRelay(logplex *url.URL, name string) (*relay, error) {
-	cfg := logplexc.Config{
-		Logplex:            *logplex,
-		HttpClient:         *http.DefaultClient,
-		RequestSizeTrigger: 100 * 1024,
-		Concurrency:        3,
-		Period:             250 * time.Millisecond,
-	}
-
-	cl, err := logplexc.NewClient(&cfg)
-	if err != nil {
-		return nil, fmt.Errorf("could not set up log channel: %v", err)
-	}
+    cfg := shuttle.NewConfig()
+    cfg.LogsURL = logplex.String()
+    cfg.Appname = "app"
+    cfg.Procid = name
+    cfg.ComputeHeader()
+
+    cl := shuttle.NewShuttle(cfg)
+    cl.Launch()
```

Comment:
@freeformz is this the correct way to configure?  when testing I still see lines with` token[shuttle] `  instead of `app[proc.number]` :(